subsys/bluetooth/common/Kconfig

```diff
@@ -100,7 +100,7 @@ config BT_BUF_ACL_RX_COUNT
 
 config BT_BUF_EVT_RX_SIZE
 	int "Maximum supported HCI Event buffer length"
-	default $(UINT8_MAX) if (BT_EXT_ADV && BT_OBSERVER) || BT_PER_ADV_SYNC || BT_DF_CONNECTION_CTE_RX || BT_CLASSIC
+	default $(UINT8_MAX) if (BT_EXT_ADV && BT_OBSERVER) || BT_PER_ADV_SYNC || BT_DF_CONNECTION_CTE_RX || BT_CLASSIC || BT_CHANNEL_SOUNDING
```

Comment:
Since this is just modifying the default: If channel sounding _need_ a specific size, you should add a `BUILD_ASSERT` in the source file to ensure this. 

Even with this change, a user can do `CONIFG_BT_BUF_EVT_RX_SIZE=68`

Reply:
I didn't find any similar BUILD_ASSERT() for the other features here that produce big hci events (extended advertising, periodic sync). Not sure what the reasoning is, I'm guessing to let advanced users save RAM ?

I guess if you're very careful you can configure channel sounding in such a way that you can assume the events will always be less than N bytes

Reply:
@olivier-le-sage is there a specific event size that shall always be supported that is larger than the default of 68?

Reply:
I'm not aware of a requirement like that.

For CS specifically the biggest events are the ones with arrayed/variable length parameters. Then, among the ones with no variable parameters, the largest is LE CS Read Remote FAE Table Complete, whose parameters take up 76 bytes

Reply:
> the largest is LE CS Read Remote FAE Table Complete, whose parameters take up 76 bytes

If that is a mandatory event to support, then you could/should set the range of `BT_BUF_EVT_RX_SIZE` to start from 76 rather than the default 68 when `BT_CHANNEL_SOUNDING` is enabled